gdal/frmts/pds/isis3dataset.cpp

```diff
@@ -1756,6 +1356,10 @@ ISIS3Dataset::ISIS3Dataset() :
     m_adfGeoTransform[3] = 0.0;
     m_adfGeoTransform[4] = 0.0;
     m_adfGeoTransform[5] = 1.0;
+
+    // Deinit JSON objects
+    m_oJSonLabel.Reset();
```

Comment:
Shouldn't the objects constructed with the default constructor be in the same state as after a Reset ?

Reply:
I didn't look at the implementation of Reset() up to now.. So my understanding is that after Reset() you can no longer do anything useful with the object, apart from assigning another valid CPLJSONObject to it? Perhaps Deinit() would actually a better name ?

Reply:
Yes, you are quite right with meaning of this function. Proposed new name looks more logical - I will rename Reset() -> Deinit().